app/client/questions/question.coffee

```diff
@@ -0,0 +1,4 @@
+Template.question.helpers
+  questionTemplateName: ->
+    data = Template.currentData()
+    return "question.#{data.question_type}"
```

Comment:
I went ahead and removed this file and the related controller since its templates were replaced by this template: https://github.com/ecohealthalliance/mobile-survey/blob/pseudo-master/app/client/questions/add_question.jade